components/builder-api/src/server/services/github.rs

```diff
@@ -1,22 +1,14 @@
 use crate::{bldr_core::{build_config::{BuildCfg,
                                        BLDR_CFG},
                         metrics::CounterMetric},
-            config::Config,
-            db::models::{account::Account,
-                         projects::Project},
             hab_core::{crypto,
                        package::{target::{self,
                                           PackageTarget},
                                  Plan}},
-            protocol::jobsrv::{JobGroup,
-                               JobGroupSpec,
-                               JobGroupTrigger},
             server::{authorize::authorize_session,
                      error::{Error,
                              Result},
-                     feat,
-                     framework::{headers,
-                                 middleware::route_message},
+                     framework::headers,
```

Comment:
you should be able to remove this whole file. We no longer respond to github events.